OpenRA.Game/Settings.cs

```diff
@@ -176,6 +176,9 @@ public class GraphicSettings
 		[Desc("Use OpenGL ES if both ES and regular OpenGL are available.")]
 		public bool PreferGLES = false;
 
+		[Desc("Choose the monitor to use in fullscreen mode.")]
```

Comment:
Style nit: Can we change this to "Display index to use in a multi-monitor fullscreen setup."?